platform/ui/src/utils/SnackbarProvider.js

```diff
@@ -1,12 +1,18 @@
-import React, { useState, createContext, useContext } from 'react';
+import React, {
+  useState,
+  createContext,
+  useContext,
+  useCallback,
+  useEffect,
+} from 'react';
 import SnackbarContainer from '../components/snackbar/SnackbarContainer';
 import SnackbarTypes from '../components/snackbar/SnackbarTypes';
 
 const SnackbarContext = createContext(null);
 
 export const useSnackbarContext = () => useContext(SnackbarContext);
 
-const SnackbarProvider = ({ children }) => {
+const SnackbarProvider = ({ children, service }) => {
```

Comment:
PropType for `service` and the one method it needs to expose?

Reply:
For sure! actually snackbar has 0 proptypes now, I should add it in this branch?

Reply:
I think that's fine :+1: